src/main/java/ch/njol/skript/SkriptCommand.java

```diff
@@ -379,7 +378,7 @@ public boolean onCommand(@Nullable CommandSender sender, @Nullable Command comma
 					TestMode.lastTestFile = script;
 				}
 				if (!script.exists()) {
```

Comment:
When using the test command, e.g. `sk test syntaxes/conditions/CondContains`, and I run `sk test` again (or just the same test again with the full command) it runs that test multiple times, once for each command previously executed:
![image](https://user-images.githubusercontent.com/29547183/180440451-2801d910-9f06-4368-84ef-94f0e6f202d3.png)

Can be fixed by unloading the script after running the tests

Reply:
For some reason `ScriptLoader.disableScripts()` is commented out, so there may have been repercussions with doing this.

Reply:
Looks like you commented this code out lol.

Re-adding it back doesn't seem to affect anything.